cmd/skopeo/delete.go

```diff
@@ -39,5 +39,14 @@ var deleteCmd = cli.Command{
 			Value: "",
 			Usage: "Use `USERNAME[:PASSWORD]` for accessing the registry",
 		},
+		cli.StringFlag{
+			Name:  "cert-path",
+			Value: "",
+			Usage: "use certificates at `PATH` (*.crt, *.cert, *.key) to connect to the registry",
+		},
+		cli.BoolTFlag{
+			Name:  "tls-verify",
+			Usage: "require HTTPS and verify certificates when talking to docker registries (defaults to true)",
+		},
```

Comment:
Repeating this all over… could we have a `credsFlagList(prefix string, kind string) []cli.Flag)` in `utils.go` containing the canonical version of these flags, and the various subcommands just concatenating that with their own private flags?

(`kind` would be something like `""`/`"source"`/"destination"` for use in the `Usage:` fields.)

That way both the flag definitions and uses would be in a single place in `utils.go`.